modules/op-release-notes-1-7.adoc

```diff
@@ -0,0 +1,307 @@
+// Module included in the following assembly:
+//
+// * cicd/pipelines/op-release-notes.adoc
+:_content-type: REFERENCE
+[id="op-release-notes-1-7_{context}"]
+= Release notes for {pipelines-title} General Availability 1.7
+
+With this update, {pipelines-title} General Availability (GA) 1.7 is available on {product-title} 4.10.
+
+[id="new-features-1-7_{context}"]
+== New features
+
+In addition to the fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.7.
+
+[id="pipelines-new-features-1-7_{context}"]
+=== Pipelines
+
+* With this update, the `when` expressions in a `Task` object are scoped to guard the tasks by default. To continue guarding the `Task` object and its dependent tasks, set the `scope-when-expressions-to-task` flag to `true`. 
++
+[NOTE]
+====
+The `scope-when-expressions-to-task` flag will be removed in a future release. As a best practice for {pipelines-shortname}, use `when` expressions scoped to the guarded `Task` only.
```

Comment:
If this flag has not been deprecated in a previous release, you might need to do now.

Reply:
Please use the standard deprecation notice format:
https://redhat-documentation.github.io/supplementary-style-guide/#release-notes